src/shared/src/components/organisms/AltinnAppHeader.test.tsx

```diff
@@ -16,6 +16,21 @@ describe('organisms/AltinnAppHeader', () => {
     partyId: '54321',
     name: 'Bedrift',
   } as IParty;
+
+  const anonymousUser = {
```

Comment:
This might be nitpicking, but I would prefer it if you could rename this to something like `selfIdentifiedUser`. We use the term `anonymous` elsewhere in the app frontend to denote a user who is _not logget in at all_, and even though this is just a variable in a test it might be confusing to use the anonymous naming here.